### ISSUE TITLE
Beginner with bxslider

### ISSUE THREAD

**Issue by FawnNyx** [stevenwanderski/bxslider-4#867]
Hello, I'm completely new to using BxSlider. I'm trying to implement it in my code and with no luck, it's just not working and I have no clue why. The pictures are just stacking ontop of each other. I am using Foundation 5 as my framework (not too sure if that has anything to do with my problem).
Any help would be much appreciated, thank you all in advance.

Here's my code:
<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    
    <title>Welcome</title>
    <link rel="stylesheet" href="css/foundation.css" />
    <link rel="stylesheet" href="css/normalize.css" />
    <link rel="stylesheet" href="css/app.css" />
    
    <script src="js/vendor/modernizr.js"></script>
   
    

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>

<script src="js/bxslider/jquery.bxslider.min.js"></script>
<script src="js/bxslider/ellybxslider.js"></script>

<link rel="stylesheet" href="js/bxslider/jquery.bxslider.css" />    
  
  </head>
  <body>
  
        
       
     
         <ul class="bxslider">
              <li>image1</li>
              <li>image2</li>
              <li>image3</li>
              <li>image4</li>
           </ul>

        
   
  
       
  
      <footer class="row">
        <div class="large-12 columns">
          <hr/>
          <div class="row">
            <div class="large-6 columns">
              <p>© 2015 Copyright </p>
            </div>
            
          </div>
        </div> 
      </footer>
    
    <script src="js/vendor/jquery.js"></script>
    <script src="js/foundation.min.js"></script>
    <script>
      $(document).foundation();
    </script>
    
     
 
   
  </body>
</html>

The actual script I have is external and looks like this:

$(document).ready(function(){
  $('.bxslider').bxSlider();
});

**Comment by Tidal-Wave** [comment]
Silly question but I don't see you including the js and css for bxslider?

**Comment by FawnNyx** [comment]
it would've helped if I didn't cut that out huh? lol 

    

 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
    <script src="jquery.bxslider.min.js"></script>
    <link href="jquery.bxslider.css" rel="stylesheet" />
    
    <script type="text/javascript">
        $(document).ready(function () {
            $(".bxslider").bxSlider();
        });
    </script>

**Comment by Tidal-Wave** [comment]
You may want to review github markdown (https://help.github.com/articles/github-flavored-markdown/) for help with commenting code pieces. You can also try making a simple example in jsfiddle. Foundation shouldn't conflict with bxslider, so you may want review a browser console for any errors.

**Comment by FawnNyx** [comment]
@Tidal-Wave Got it! thank you! Please let me know if it still isn't showing.

**Comment by Tidal-Wave** [comment]
So are there any errors in the browser console?

**Comment by FawnNyx** [comment]
@Tidal-Wave Man, I've been trying to figure it out, and still no dice. I apologize that I didn't respond sooner.
I just checked my browser console for errors and nothing is popping up.
I took your suggestion and plopped my code in to JSfiddle:
https://jsfiddle.net/Fawn_Nyx/38ec0er0/10/

**Comment by Tidal-Wave** [comment]
so, couple of things.. 

you have multiple errors in the console, including some 404s. You have two copies of jQuery, which is a no-no, and it doesn't seem to want to pull your copy of bxslider remotely into jsfiddle.

I added the bxslider js file via rawgit and everything worked fine: https://jsfiddle.net/dq2dq8py/

**Comment by FawnNyx** [comment]
Wow thank you! 
I saw the errors....its another thing I'm trying to learn how to work with...trying to figure out what it all means. Its all alien to me.

I appreciate your help I'm learning a lot!
